akamai/resource_akamai_property_activation.go

```diff
@@ -174,12 +174,22 @@ func resourcePropertyActivationExists(d *schema.ResourceData, meta interface{})
 
 	network := papi.NetworkValue(d.Get("network").(string))
 	version := d.Get("version").(int)
+
```

Comment:
The property activation changes are a bit more questionable.  There are a couple minor tweaks for comments and a few questions i will put below.